src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+  margin-top: 2px;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin: 0 2px;
```

```suggestion
  margin: 0 2px;
```
distance between stars should be 4px for all the stars except the last one